hibp.js

```diff
@@ -64,16 +62,35 @@ const HIBP = {
     return await this._throttledGot(url, reqOptions);
   },
 
+  // make list of all PNG logos
+  makeLogoListForEmails() {
+    const emailLogos = [];
+    const images = fs.readdirSync("./public/img/logos/");
+    images.forEach(file => {
+      if (file.endsWith("png")) {
+        emailLogos.push(file);
+      }
+    });
+    return emailLogos;
+  },
+
   async loadBreachesIntoApp(app) {
     log.info("loadBreachesIntoApp");
     try {
+      const pngLogoList = await this.makeLogoListForEmails();
       const breachesResponse = await this.req("/breaches");
       const breaches = [];
 
       for (const breach of breachesResponse.body) {
         // const breach = breachesResponse.body[breachIndex];
-        // purify the description
-        breach.Description = DOMPurify.sanitize(breach.Description, {ALLOWED_TAGS: []});
+
+        // check if we have png logo & set breach.ImgSrc to "missing-logo-icon" if not.
+        if (!pngLogoList.includes(`${breach.Name}.png`)) {
+          log.info(`missing image ${breach.Name}`);
+          breach.ImgSrc = "missing-logo-icon";
+        } else {
+          breach.ImgSrc = `${breach.Name}`;
```

Comment:
nit: just use `breach.ImgSrc = breach.Name;`. I don't think it needs interpolation.